### DISCUSSION TITLE
Request: "Compact View" for Meal Plan

### DISCUSSION THREAD

**Discussion by ensjens** [TandoorRecipes/recipes#2815] [Ideas]
Each recipe's picture takes up a lot of space, so if there are multiple recipes planned in a day, you cannot see all of the recipes at a glance and you need to scroll down. It would be helpful to have a "compact view" without the pictures so that more recipes can fit in one window.

**Comment by vabene1111** [comment]
which version are you on?

**Reply by ensjens** [reply]
ah, I was on 1.5.6. I upgraded to 1.5.11 and it's looking much better now! Thanks for all your work on this!
